### DISCUSSION TITLE
Trying to migrate the Core WebApi project from .Net core 2.1 to .Net 8.0

### DISCUSSION THREAD

**Discussion by sibinthomask** [domaindrivendev/Swashbuckle.AspNetCore#3200] [General]
I am trying to migrate the Core WebApi project from .Net core 2.1 to .Net 8.0. As we do usually, I tried to add packages from NuGet. To solve errors, I installed Swashbuckle,Swashbuckle.AspNetCore,Swashbuckle.AspNetCore.SwaggerGen,Swashbuckle.AspNetCore.SwaggerUI,Swashbuckle.AspNetCore.Filters.

Now I am getting this error -
TypeLoadException: Could not load type 'Swashbuckle.AspNetCore.Swagger.Info' from assembly 'Swashbuckle.AspNetCore.Swagger, Version=7.2.0.0, Culture=neutral, PublicKeyToken=62657d7474907593'.

Please share your ideas to solve this error.

**Comment by martincostello** [comment]
Please share an entire project as a repository that demonstrates the project as requested in the issue as requested earlier.

Please stop creating duplicate issues or discussions as it dilutes the problem across multiple places.

Without a self contained project that we can checkout so we can debug we will not be able to help you.
